application/Commands/Types/Relationship.php

```diff
@@ -328,4 +328,68 @@ protected function delete_item( $slug, $do_cleanup = true ) {
 
 		return true;
 	}
-}
+
+	/**
+	 * Displays posts related to a given post.
+	 *
+	 * ## OPTIONS
+	 *
+	 * --post=<number>
+	 * : The ID of the post.
+	 *
+	 *	--relationship=<string>
+	 * : The relationship slug.
+	 *
+	 * [--role-to-return=<string>]
+	 * : The role to return.
+	 *
+	 * [--query-by-role=<string>]
+	 * : The role to query by.
+	 *
+	 * [--format=<format>]
+	 * : The format of the output. Can take values: table, csv, json, count, yaml. Default: table.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *    wp types relationship get-related-posts --post=123 --relationship=slug
+	 *    wp types relationship get-related-posts --post=123 --relationship=slug --role-to-return=child --query-by-role=parent
+	 *    wp types relationship get-related-posts --post=123 --relationship=slug --role-to-return=child --query-by-role=parent --format=json
+	 *
+	 * @subcommand get-related-posts
+	 * @synopsis --post=<number> --relationship=<string> [--role-to-return=<string>] [--query-by-role=<string>] [--format=<format>]
+	 * @link https://toolset.com/documentation/customizing-sites-using-php/post-relationships-api/#toolset_get_related_posts
+	 */
+	public function get_related_posts( $args, $assoc_args ) {
```

Comment:
Could you please rename (and move) this so that the command is `wp types association query`?

The "relationship" subcommand is meant for querying or manipulating relationship _definitions_, while the "association" subcommand deals with individual connections between posts.

Reply:
Will do!